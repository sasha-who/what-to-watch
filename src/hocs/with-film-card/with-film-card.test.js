import * as React from "react";
import renderer from "react-test-renderer";
import withFilmCard from "./with-film-card.js";

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withFilmCard(MockComponent);

it(`withFilmCard should render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
