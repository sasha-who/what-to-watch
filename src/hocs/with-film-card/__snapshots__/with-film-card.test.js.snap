// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withFilmCard should render correctly 1`] = `<div />`;
